src/main/java/com/faforever/client/audio/AudioServiceImpl.java

```diff
@@ -142,7 +142,8 @@ public void playFriendJoinsGameSound() {
     if (!notificationsPrefs.isFriendJoinsGameSoundEnabled()) {
       return;
     }
-    playSound(friendJoinsGameSound);
+    // FIXME implement
```

Comment:
Why exactly did you delete it?

Reply:
```
//    friendJoinsGameSound = loadSound(FRIEND_JOINS_GAME_SOUND);
```

That sound doesn't exist. I figured - since friendOffline, friendOnline, and friendPlaysGame are also sounds not implemented yet - it was either a work in progress or someone decided to not use the same sound as the others that are already implemented.

Reply:
Hmm ok

Reply:
Would you make an issue on that, if there is no one yet ;)

Reply:
issue made: #746